src/UI/Icon.elm

```diff
@@ -443,6 +445,26 @@ remove hint =
     Icon (Properties hint Remove) defaultOptions
 
 
+{-| A trash can with an "x" on it.
+
+    Icon.delete "Delete item"
+
+-}
+delete : String -> Icon
```

Comment:
check this Item because I cant see it in the showcase

Reply:
oops find n replace gone wrong